pilot/helpers/agents/Developer.py

```diff
@@ -682,9 +682,11 @@ def continue_development(self, iteration_convo, last_branch_name, continue_descr
         tried_alternative_solutions_to_current_issue = self.project.last_iteration['prompt_data']['tried_alternative_solutions_to_current_issue'] if (self.project.last_iteration and 'tried_alternative_solutions_to_current_issue' in self.project.last_iteration['prompt_data']) else []
         next_solution_to_try = None
         iteration_count = self.project.last_iteration['prompt_data']['iteration_count'] if (self.project.last_iteration and 'iteration_count' in self.project.last_iteration['prompt_data']) else 0
+        # should_review is used to block review if CLI user input is "r" (run command) and we after execution we go back to while loop
+        should_review = True
         while True:
             self.user_feedback = llm_solutions[-1]['user_feedback'] if len(llm_solutions) > 0 else None
-            review_successful = self.project.skip_steps or self.review_task()
+            review_successful = self.project.skip_steps or not should_review or (should_review and self.review_task())
```

Comment:
just to make sure I understand, this is fixing a different unrelated bug (reviewer being triggered after command run)?